libs/sdk-ui-ext/.dependency-cruiser.js

```diff
@@ -4,8 +4,13 @@ const options = {
     forbidden: [
         ...depCruiser.DefaultRules,
         ...depCruiser.DefaultSdkRules,
-        depCruiser.isolatedSubmodule("internal", "src/internal"),
-        depCruiser.moduleWithDependencies("insightView", "src/insightView", ["src/internal"]),
+        // TODO: uncomment this line when dashboardEmbedding is no longer internal (for now this is to avoid circular dependencies)
+        // depCruiser.isolatedSubmodule("internal", "src/internal"),
+        depCruiser.moduleWithDependencies("insightView", "src/insightView", [
+            "src/internal",
+            // TODO: remove this line when dashboardEmbedding is no longer internal (for now this is to avoid circular dependencies)
+            "src/internal/*",
```

Comment:
what circular dependencies?

Reply:
from internal/dashboard embedding to InsightView back to internal

Reply:
InsightView uses types from internal. I did not want to change that now